app-emulation/qemu/qemu-7.2.0-r1.ebuild

```diff
@@ -176,11 +176,14 @@ SOFTMMU_TOOLS_DEPEND="
 	fuse? ( >=sys-fs/fuse-3.1:3[static-libs(+)] )
 	glusterfs? ( >=sys-cluster/glusterfs-3.4.0[static-libs(+)] )
 	gnutls? (
-		dev-libs/nettle:=[static-libs(+)]
 		>=net-libs/gnutls-3.0:=[static-libs(+)]
+		dev-libs/nettle:=[static-libs(+)]
 	)
 	gtk? (
+		x11-libs/cairo
+		x11-libs/gdk-pixbuf:2
 		x11-libs/gtk+:3
+		x11-libs/libX11
```

Comment:
I'm not 100% sure if this is what's desirable here, due to different deps required for X/wayland. Maybe we should switch `USE=gtk` for a separate `USE=wayland` and `USE=X`?